docs/docs/documentation/getting-started/authentication/oidc-v2.md

```diff
@@ -10,7 +10,7 @@
 Mealie supports 3rd party authentication via [OpenID Connect (OIDC)](https://openid.net/connect/), an identity layer built on top of OAuth2. OIDC is supported by many Identity Providers (IdP), including:
 
 - [Authentik](https://goauthentik.io/integrations/sources/oauth/#openid-connect)
-- [Authelia](https://www.authelia.com/configuration/identity-providers/open-id-connect/)
+- [Authelia](https://www.authelia.com/configuration/identity-providers/open-id-connect/mealie/)
```

Comment:
I tried this URL and it doesn't seem to work; did you mean this one?

```suggestion
- [Authelia](https://www.authelia.com/integration/openid-connect/mealie/)
```

Reply:
Thanks! Was doing too many things and butchered it.